tests/phpunit/elementor/core/breakpoints/test-manager.php

```diff
@@ -0,0 +1,157 @@
+		$breakpoints_default_config = Breakpoints_Manager::get_default_config();
+
+		$has_custom_breakpoints = false;
+
+		foreach ( $active_breakpoints as $breakpoint_name => $breakpoint ) {
+			if ( $breakpoints_default_config[ $breakpoint_name ]['default_value'] !== $breakpoint->get_value() ) {
+				$has_custom_breakpoints = true;
+				break;
+			}
+		}
+
+		$this->assertEquals( $has_custom_breakpoints, Plugin::$instance->breakpoints->has_custom_breakpoints() );
+	}
+
+	/**
+	 * Test Get Device Min Breakpoint
+	 *
+	 * Test the Breakpoints_Manager::get_device_min_breakpoint() method.
+	 * This test only tests the default breakpoints (mobile, tablet).
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_device_min_breakpoint() {
+		$active_breakpoints = Plugin::$instance->breakpoints->get_active_breakpoints();
+
+		$this->assertEquals( $active_breakpoints[ Breakpoints_Manager::BREAKPOINT_KEY_MOBILE ]->get_value() + 1, Plugin::$instance->breakpoints->get_device_min_breakpoint( Breakpoints_Manager::BREAKPOINT_KEY_TABLET ) );
+	}
+
+	/**
+	 * Test Get Device Min Breakpoint - Mobile
+	 *
+	 * Test the Breakpoints_Manager::get_device_min_breakpoint() method for the mobile device case, which is a special
+	 * case.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_device_min_breakpoint_mobile() {
+		$active_breakpoints = Plugin::$instance->breakpoints->get_active_breakpoints();
+
+		// Test for mobile specifically, which always has a min point of 0.
+		$this->assertEquals( 0, Plugin::$instance->breakpoints->get_device_min_breakpoint( Breakpoints_Manager::BREAKPOINT_KEY_MOBILE ) );
+	}
+
+	/**
+	 * Test Get Desktop Min Breakpoint
+	 *
+	 * Test the Breakpoints_Manager::get_desktop_min_point() method with default active breakpoints.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_desktop_min_point() {
+		$tablet_breakpoint = Plugin::$instance->breakpoints->get_breakpoints( 'tablet' );
+
+		$this->assertEquals( $tablet_breakpoint->get_value() + 1, Plugin::$instance->breakpoints->get_desktop_min_point() );
+	}
+
+	/**
+	 * Test Get Desktop Min Breakpoint When Laptop Enabled
+	 *
+	 * Test the Breakpoints_Manager::get_desktop_min_point() method when the laptop breakpoint is enabled.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_desktop_min_point_when_laptop_enabled() {
+		$this->set_admin_user();
+
+		$kit = Plugin::$instance->kits_manager->get_active_kit();
+		$kit_settings = $kit->get_settings();
+
+		// Add the laptop breakpoint to the active breakpoints setting.
+		$kit_settings['active_breakpoints'][] = 'viewport_laptop';
+
+		// Save kit settings.
+		$kit->save( [ 'settings' => $kit_settings ] );
+
+		// Refresh kit.
+		$kit = Plugin::$instance->documents->get( $kit->get_id(), false );
+
+		// The test is done on a new manager, because the instance attached to the Plugin instance already set its
+		// active breakpoints, and this setting cannot be changed once it is set.
+		$manager = new Breakpoints_Manager();
```

Comment:
Instead of that, create a refresh method inside the manager and call it after saving breakpoints settings to the kit

Reply:
Done.